### PR TITLE
Syn 288 Added ZohoInventoryAccessCredentialAPI

### PR DESCRIPTION
**Pull request by hapanivivek**
Added post method with getAccessCredentials

### CODE REVIEW

File: src/main/java/io/github/dftrakesh/zoho/inventory/ZohoInventorySdk.java [hunk 1]
```diff
@@ -95,6 +95,13 @@ protected HttpRequest post(URI uri, UpdateRecordRequest updateRecordData) {
             .headers(AUTHORIZATION_HEADER, TOKEN_NAME.concat(accessCredential.getAccessToken()))
             .build();
     }
+    
+    protected HttpRequest postWithNoBody(URI uri) {
```
Comment (manan-dft) [inline src/main/java/io/github/dftrakesh/zoho/inventory/ZohoInventorySdk.java]: Name it postWithOutBody

File: src/main/java/io/github/dftrakesh/zoho/inventory/ZohoInventorySdk.java [hunk 2]
```diff
@@ -95,6 +95,13 @@ protected HttpRequest post(URI uri, UpdateRecordRequest updateRecordData) {
...
+        return HttpRequest.newBuilder(uri)
+                .header("Content-Type", "application/json")
```
Comment (manan-dft) [inline src/main/java/io/github/dftrakesh/zoho/inventory/ZohoInventorySdk.java]: We have the constant for Content-Type?

**Review by manan-dft — CHANGES_REQUESTED** [review]
Complete the commented changes

File: src/main/java/io/github/dftrakesh/zoho/inventory/ZohoInventorySdk.java (hunk 1 above)
Comment (manan-dft) [reply src/main/java/io/github/dftrakesh/zoho/inventory/ZohoInventorySdk.java]: Also for "application/json"

Comment (hapanivivek) [reply src/main/java/io/github/dftrakesh/zoho/inventory/ZohoInventorySdk.java]: Added Constant and changed name.

File: src/main/java/io/github/dftrakesh/zoho/inventory/ZohoInventorySdk.java (hunk 2 above)
Comment (hapanivivek) [reply src/main/java/io/github/dftrakesh/zoho/inventory/ZohoInventorySdk.java]: Changed and used Constant

**Review by manan-dft — APPROVED** [review]

**Pull request merged**